.changeset/blue-melons-stay.md

```diff
@@ -0,0 +1,36 @@
+---
+"@biomejs/biome": minor
+---
+
+Enhanced the command `migrate eslint`. Now the command shows which ESLint rules were migrated,
+and which rules aren't supported yet.
+
+```
+/eslint.config.js migrate ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ℹ 50% (3/6) of the rules have been migrated.
+
+  ℹ Migrated rules:
+
+  - getter-return
+  - prefer-const
+  - @typescript-eslint/require-await
+
+  ℹ Rules that can be migrated to an inspired rule using --include-inspired:
+
+  - @typescript-eslint/parameter-properties
+
+  ℹ Rules that can be migrated to a nursery rule using --include-nursery:
+
+  - @typescript-eslint/switch-exhaustiveness-check
+
+  ℹ Unsupported rules:
+
+  - semi
```

Comment:
Suggestion: It would be really cool if we also had a list of rules that are covered by formatter functionality.

Reply:
What do you mean?

Reply:
Meaning that `semi` is handled by our formatter. So if somewhere we have a list of stylistic rules, we can say "hey, this rule is taken care by our formatter using this configuration"